mta-archive-builder/README.md

```diff
@@ -0,0 +1,42 @@
+# Multitarget Application Archive Builder
+The multitarget application archive builder is a command-line tool that packages a multitarget application into a deployable archive (MTAR). For full documentation please visit the [SAP Help Portal](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html).
+
+This image can be used to build SAP Multitarget Applications (MTA) containing Java and Node.js modules.
+
+# How to use this image
+
+On a linux machine you can run 
+
+`docker run --rm -v `pwd`:/project -it mta-archive-builder:latest mtaBuild --version`
+
+This will execute the MTA archive builder and print its version information.
+
+`docker run --rm -v `pwd`:/project -it mta-archive-builder:latest mtaBuild --mtar dummy.mtar --build-target NEO build`
```

Comment:
Not sure about the `:latest`, in my opinion its an antipattern.
```suggestion
`docker run --rm -v `pwd`:/project mta-archive-builder mtaBuild --mtar dummy.mtar --build-target NEO build`
```

Reply:
IMO `<imageName>` and `<imageName>:latest` are fully equivalent. Having the `:latest` is more transparent.